domain/src/main/java/edu/nextstep/camp/calculator/domain/util/CalculatorInputUtil.kt

```diff
@@ -0,0 +1,14 @@
+package edu.nextstep.camp.calculator.domain
+
+class CalculatorInputUtil {
+    companion object {
+        fun isNumberRegex(text: String): Boolean {
+            var regex = Regex("[0-9]")
```

Comment:
```suggestion
            val regex = Regex("[0-9]")
```
변수를 정의하고나서 변수의 값을 변경할 일이 없다면, val을 활용해주시면 좋을 것 같아요. (이하 관련 내용 동일)